### ISSUE TITLE
ValueError: Missing type index for text

### ISSUE THREAD

**Issue by letmejustputthishere** [rocklabs-io/ic-py#66]
I have the following `did` file
```
type Canister = 
  service {
    tokens: (AccountIdentifier__1) -> (Result_1) query;
 };
type CommonError = 
  variant {
   InvalidToken: TokenIdentifier;
   Other: text;
};
type AccountIdentifier__1 = text;
type TokenIndex = nat32;
type TokenIdentifier = text;
type Result_1 = 
 variant {
   err: CommonError;
   ok: vec TokenIndex;
 };
service : () -> Canister
```

I setup the canister like this
```
# create agent
iden = Identity()
client = Client()
agent = Agent(iden, client)

ext_did = open("ext.did").read()

ext = Canister(
    agent, canister['canisterId'], candid=ext_did)
```

But making a call to `ext.tokens(account)` fails with the error mentioned above. Is this a bug or am I doing something wrong?

**Comment by Myse1f** [comment]
fix in #67

**Comment by letmejustputthishere** [comment]
are you expecting use to close issues that have been resolved ourselves? or are you waiting for some internal process before you close them? i see a couple of issues that have been resolved but they are still open
